policy/modules/contrib/tuned.te

```diff
@@ -41,7 +41,7 @@ allow tuned_t self:netlink_socket create_socket_perms;
 allow tuned_t self:udp_socket create_socket_perms;
 allow tuned_t self:socket create_socket_perms;
 
-read_files_pattern(tuned_t, tuned_etc_t, tuned_etc_t)
+rw_files_pattern(tuned_t, tuned_etc_t, tuned_etc_t)
```

Comment:
I can see `tuned_rw_etc_t` below, which seems to be intended for rw files. So I believe we should rather change the label for the file(s) that tuned needs to write to.

Reply:
Looks safe, the only interface making tuned_rw_etc_t accessible is tuned_admin. So I found all 4 instances of changed files, and while there, fixed the incorrect existing transition.